website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -1,197 +1,173 @@
+  )
+}}
+```
 
-Controls the save experience of an edited cell.
-**Row level** - Cells can be saved using the Save/Discard column buttons\
-**Custom** - Cells can be saved by using an onSubmit action of the column or through an external button.
+Now you are ready to edit your table in any number of rows, and the "Save All" button should submit all your changes to the datasource and refresh the table.
 
-### Cell Wrapping
+#### Cell wrapping
 
-Controls how overflowing contents of the column are handled.  When turned on the Contents get wrapped to the next line.
+Controls how overflowing contents of the column are handled. When turned on, the contents get wrapped to the next line.
 
-### Editable
+#### Editable
 
 Controls whether cells of the column are editable. To make the column editable, click the checkbox inside a column card that makes all the cells in that column editable.
 
-Alternatively, A column can be editable by turning on `Editable` property inside column settings.
-
+Alternatively, a column can be made editable by turning on the `Editable` switch inside that column's settings.
 
 <VideoEmbed host="youtube" videoId="BW5cVp0GfLE" title="Making column editable" caption="Making column editable"/>
 
-
 :::info
-Validation ensures that only certain values are entered in a field. You have to make the column editable to allow validation. The validation feature is available for **text** and **numeric** column types.
+You can implement user input validation to ensure that only certain values are entered in a cell. The validation features require the column to be editable, and the validation options can be found in a **text** or **numeric** column's settings.
 :::
 
-### Regex
+#### Regex
 
-Regex adds validation to the cell value which displays an error on failure. When the input doesn't match the regex expression, the input is considered invalid. Using `Regex` or Regular expression property, you can set specific constraints on the input you expect from the user.
+[Regular expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions) (regex) are patterns that describe valid user input. For example, the pattern `[a-zA-Z]` describes a string that contains only letters of the English alphabet. Using regular expressions in the **Regex** property of columns, you can set specific constraints on the input you expect from the user.
 
-For example, add a regular expression for entering a name. The name can contain only alphabets and spaces between the first and last name.
+When you add a regular expression to a column, all user input in that column is compared to the pattern; it's considered to be valid when it matches the expression, or invalid when it doesn't match. When input is invalid, the cell displays its **Error message** to the user in a tool-tip.
+
+For example, add a regular expression for entering a name. The name can contain only alphabetical characters and spaces between the first and last name:
```

Comment:
Adding a screenshot might help here. Any one screenshot showing an error message in a tooltip should be good.